pkgs/development/python-modules/git-revise/default.nix

```
@@ -29,6 +30,11 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
+  disabledTests = lib.optionals stdenv.isDarwin [
```

Should be
```suggestion
  disabledTests = lib.optionals stdenv.hostPlatform.isDarwin [
```
these days. (And I guess the dependency should be conditional too.)